README.md

```diff
@@ -5,61 +5,103 @@
 [![Total Downloads](https://img.shields.io/packagist/dt/muffin/obfuscate.svg?style=flat-square)](https://packagist.org/packages/muffin/obfuscate)
 [![License](https://img.shields.io/badge/license-MIT-blue.svg?style=flat-square)](LICENSE)
 
-CakePHP 3 support for ID obfuscation using any combination of HashIds, Optimus and/or Tiny.
+Primary key obfuscation for CakePHP 3 using any combination of
```

Comment:
As far as this heading text is concerned I could change it to something more readable like "using X, Y and/or custom strategies".

In general, it's exactly the fact that you can mix and match as needed that makes this plugin awesome and.. is actually how I use it. Optimus for user-ids, Tiny for "linkable" resources.


Sent from my Mobile Device

> Op 10 mrt. 2016 om 12:31 heeft ADmad <notifications@github.com> het volgende geschreven:
> 
> In README.md:
> 
> > @@ -5,61 +5,103 @@
> >  [![Total Downloads](https://img.shields.io/packagist/dt/muffin/obfuscate.svg?style=flat-square)](https://packagist.org/packages/muffin/obfuscate)
> >  [![License](https://img.shields.io/badge/license-MIT-blue.svg?style=flat-square)](LICENSE)
> >  
> > -CakePHP 3 support for ID obfuscation using any combination of HashIds, Optimus and/or Tiny.
> > +Primary key obfuscation for CakePHP 3 using any combination of
> I am not too sure about using the word "combination". Most people would only use either one of them.
> 
> —
> Reply to this email directly or view it on GitHub.
>